src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1617,19 +1777,21 @@ private void hitBlockWithToolInHand(@Nullable final BlockPos blockPos, final boo
             if (!worldObj.isRemote)
             {
                 MineColonies.getNetwork().sendToAllAround(
-                  //todo: correct side
-                  new BlockParticleEffectMessage(blockPos, worldObj.getBlockState(blockPos), 1),
-                  new NetworkRegistry.TargetPoint(worldObj.provider.getDimension(), blockPos.getX(), blockPos.getY(), blockPos.getZ(), BLOCK_BREAK_PARTICLE_RANGE));
+                        //todo: correct side
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)